examples/example.c

```diff
@@ -23,7 +23,7 @@ int main(void)
     sfSprite_setPosition(sprite, spritePosition);
 
     // Create a graphical text to display
-    sfFont* font = sfFont_createFromFile("tuffy.ttf");
```

Comment:
Adding `const` here causes a compiler error on CSFML `master` as it stands today. There's no good reason to prevent this.

Reply:
yea we don't do the same mistake as C free.